tools/csv-merger/csv-merger.go

```diff
@@ -180,7 +182,8 @@ func validateNoApiOverlap(crdDir string) bool {
 				if stringInSlice(apigroup, crdMap[comparedOperator]) {
 					overlappingOperators := []string{operator, comparedOperator}
 					for _, o := range overlappingOperators {
-						if !stringInSlice(o, overlapsMap[apigroup]) {
+						// We work on replacement for current v2v. Remove this check when vmware import is removed
+						if !stringInSlice(o, overlapsMap[apigroup]) && apigroup != "v2v.kubevirt.io" {
```

Comment:
If your operator is using the same API group (`v2v.kubevirt.io`) as an existing CRD deployed by other operator, please remove the old obsolete one, instead of excluding `v2v.kubevirt.io` from the test.
Thanks

Reply:
Old operator would be still working for some time since the new operator is not yet capable to do the same. Hopefully the switch will happen in the next version.